### PR TITLE
footer improved

### PR DESCRIPTION
**Pull request by davidkreidler**

### CODE REVIEW

**Comment by Langleu** [comment]
wolke
Idee ganz nett, aber glaube du musst die Wolke nochmal überarbeiten bzgl. des Kontrastes

**Comment by MasterCarl** [comment]
I think it would be nice to have just the legally required minimum (Impressum) on application pages, but a more complete footer would be nice for the landing page

**Comment by NicoKnoll** [comment]
bildschirmfoto 2016-11-30 um 11 42 52

Did you even look at it or was it just copying some hind of code here? We don't have youtube or twitter so why are there links for it? And why is it overlaying the image? I'd rather have no footer for now than one that looks like a bug.

**Comment by nicoknoll** [comment]
@davidkreidler Could you post an image on how it looks now?

File: src/modules/static/styles/footer.scss [hunk 1]
```diff
@@ -1,49 +1,17 @@
 footer {
 	width:100%;
-	/*border-top:2px solid #727272;*/
+	border-top:2px solid #727272;
```
Comment (nicoknoll) [inline src/modules/static/styles/footer.scss]: No border top. Instead add "margin-top: 40px; "

File: src/modules/static/components/footer.jsx [hunk 2]
```diff
@@ -11,20 +11,16 @@ class Footer extends React.Component {
...
-						</div>
-						<div className="bg-graphic bg-graphic-footer"></div>
-					</div>
 					<div className="center">
 						<img src="https://hpi.de/fileadmin/templates/img/hpi_header_logo_pos_srgb.png"></img>
 						<a>2016 Schul-Cloud </a><i className="fa fa-cloud" aria-hidden="true"></i>
+						<a>&emsp;</a>
+						<nobr>
+							<a href="https://hpi.de/impressum.html" target="blank"><i className="fa fa-info" aria-hidden="true"></i> Impressum </a>
+							<a href="mailto:hpi-info@hpi.de?subject=Schul_Cloud%20Anfrage"><i className="fa fa-at" aria-hidden="true"></i> Kontakt</a>
```
Comment (nicoknoll) [inline src/modules/static/components/footer.jsx]: No icons

File: src/modules/static/components/footer.jsx [hunk 3]
```diff
@@ -11,20 +11,16 @@ class Footer extends React.Component {
...
-						<a href="mailto:hpi-info@hpi.de?subject=Schul_Cloud%20Anfrage"><i className="fa fa-at" aria-hidden="true"></i> Kontakt</a>
-						</div>
-						<div className="bg-graphic bg-graphic-footer"></div>
-					</div>
 					<div className="center">
 						<img src="https://hpi.de/fileadmin/templates/img/hpi_header_logo_pos_srgb.png"></img>
 						<a>2016 Schul-Cloud </a><i className="fa fa-cloud" aria-hidden="true"></i>
+						<a>&emsp;</a>
+						<nobr>
+							<a href="https://hpi.de/impressum.html" target="blank"><i className="fa fa-info" aria-hidden="true"></i> Impressum </a>
```
Comment (nicoknoll) [inline src/modules/static/components/footer.jsx]: no icons (remove "<i className="fa fa-info" aria-hidden="true"></i>")

File: src/modules/static/components/footer.jsx [hunk 4]
```diff
@@ -11,20 +11,16 @@ class Footer extends React.Component {
...
-						<div className="right">
-						<a href="https://hpi.de/impressum.html" target="blank"><i className="fa fa-info" aria-hidden="true"></i> Impressum</a><br/>
-						<a href="mailto:hpi-info@hpi.de?subject=Schul_Cloud%20Anfrage"><i className="fa fa-at" aria-hidden="true"></i> Kontakt</a>
-						</div>
-						<div className="bg-graphic bg-graphic-footer"></div>
-					</div>
 					<div className="center">
 						<img src="https://hpi.de/fileadmin/templates/img/hpi_header_logo_pos_srgb.png"></img>
 						<a>2016 Schul-Cloud </a><i className="fa fa-cloud" aria-hidden="true"></i>
+						<a>&emsp;</a>
```
Comment (nicoknoll) [inline src/modules/static/components/footer.jsx]: Nope - a elements are for links. What is the purpose of having an empty link here?

File: src/modules/static/components/footer.jsx [hunk 5]
```diff
@@ -11,20 +11,16 @@ class Footer extends React.Component {
...
 				<div className="imprint">
-					<div className="container-fluid">
-						<div className="right">
-						<a href="https://hpi.de/impressum.html" target="blank"><i className="fa fa-info" aria-hidden="true"></i> Impressum</a><br/>
-						<a href="mailto:hpi-info@hpi.de?subject=Schul_Cloud%20Anfrage"><i className="fa fa-at" aria-hidden="true"></i> Kontakt</a>
-						</div>
-						<div className="bg-graphic bg-graphic-footer"></div>
-					</div>
 					<div className="center">
 						<img src="https://hpi.de/fileadmin/templates/img/hpi_header_logo_pos_srgb.png"></img>
```
Comment (nicoknoll) [inline src/modules/static/components/footer.jsx]: Don't use static images from other servers as their paths could change. Please add the image to this repo directly and use relativ paths.

File: src/modules/static/components/footer.jsx [hunk 6]
```diff
@@ -11,20 +11,16 @@ class Footer extends React.Component {
...
 						<a>2016 Schul-Cloud </a><i className="fa fa-cloud" aria-hidden="true"></i>
```
Comment (nicoknoll) [inline src/modules/static/components/footer.jsx]: Don't hard code the 2016. Use something like: {new Date().getFullYear()}

Comment (nicoknoll) [inline src/modules/static/components/footer.jsx]: And no icons

**Comment by davidkreidler** [comment]
image

**Pull request merged**